g3doc/index.md

```diff
@@ -833,13 +833,13 @@ Swift:
 
 ```swift
 func work1(_ string: String) -> Promise<String> {
-  return Promise {
+  return Promise { () -> String in
```

Comment:
Is this needed? Copying and pasting this code snipped into Promises.xcworkspace --> Playground seems to run fine on Xcode 10.1.

Reply:
@temrich agree, the code works without additional statements. Just wanted to be consistent with what #84 introduced. 

Feel freet close this PR.

Reply:
In cases where compiler can infer the type, we've elected to not be explicit in the documentation. But if I remember correctly, the instances in #84 were required.